### PR TITLE
Add a reinforced barrel to the Flak Cannon's recipe

### PR DESCRIPTION
**Pull request by ItsSamGray**
## Changes

- What it says on the tin

## Reasoning

- Added the reinforced barrel to the recipe just like how the vanilla mortar is set up for consistency's sake and reducing steel cost bloat.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [ ] Game runs without errors

### CODE REVIEW

**Review by N7Huntsman — COMMENTED** [review]
The reduced steel cost + barrel requirement seems reasonable to me. Will have to test the difficulty variance bit, as that's new to me, but it looks fine.

**Comment by ItsSamGray** [comment]
The same PR for the 105mm Howitzer is also opened

**Review by albertbokor — APPROVED** [review]

File: Defs/ThingDefs_Buildings/Buildings_Turrets.xml [hunk 1]
```diff
@@ -377,10 +377,15 @@
...
-      <ComponentIndustrial>5</ComponentIndustrial>
-    </costList>
+    <costListForDifficulty>
+      <difficultyVar>classicMortars</difficultyVar>
+      <invert>true</invert>
+      <costList>
+        <ComponentIndustrial>5</ComponentIndustrial>
+        <ReinforcedBarrel>1</ReinforcedBarrel>
+      </costList>
+      <Steel>350</Steel>
```
Comment (N7Huntsman) [inline Defs/ThingDefs_Buildings/Buildings_Turrets.xml:387]: This throws a start-up error. I'm still looking into the structure of how exactly the structure of this tag works, so I'm not sure what the correct format is.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

**Comment by N7Huntsman** [comment]
Submitted #1553 with the correct xml format.

**Pull request closed without merging**